homeassistant/components/sabnzbd/quality_scale.yaml

```diff
@@ -0,0 +1,81 @@
+rules:
+  # Bronze
+  action-setup: done
```

Comment:
We're unloading the services in `async_unload_entry`, we should keep them. (If we should remove them after all entries of that component are gone, that should be a concern for the core imo)